webapp/sources/rudder/rudder-web/src/main/webapp/secure/administration/setup.html

```diff
@@ -0,0 +1,28 @@
+<lift:surround with="onboarding-layout" at="content">
```

Comment:
I don't think I understand why we need a different layout ?

(and in all cases, it's not the display we want: 
![image](https://user-images.githubusercontent.com/44649/112617012-b5808e00-8e24-11eb-8c16-33e240f795a5.png)

Also, there's a js error on that page, not sure if related: 

```
Uncaught TypeError: $.fn.dataTable is undefined
    <anonymous> http://localhost:8082/rudder-web/cache-7.0.0~alpha1-SNAPSHOT/javascript/rudder/rudder.js:462
    jQuery 13
```